packages/firestore/test/unit/specs/query_spec.test.ts

```diff
@@ -100,6 +100,27 @@ describeSpec('Queries:', [], () => {
       });
   });
 
+  Caused by: java.lang.AssertionError: expected:<
```

Comment:
what is it?

Reply:
This .... ehmm... seems unrelated. Removed.